client/my-sites/earn/memberships/index.jsx

```diff
@@ -528,60 +533,82 @@ class MembershipsSection extends Component {
 	}
 
 	renderOnboarding( cta ) {
-		const isPayments = isEnabled( 'earn/rename-payment-blocks' );
+		const { translate } = this.props;
+
 		return (
-			<div className="memberships__onboarding-wrapper">
-				<div className="memberships__onboarding-column-info">
-					<div className="memberships__onboarding-header">
-						{ isPayments
-							? this.props.translate( 'Introducing Payments.' )
-							: this.props.translate( 'Introducing Recurring Payments.' ) }
+			<Card>
+				<div className="memberships__onboarding-wrapper">
+					<div className="memberships__onboarding-column-info">
+						<h2 className="memberships__onboarding-header">
+							{ preventWidows( translate( 'Accept payments on your website.' ) ) }
+						</h2>
+						<p className="memberships__onboarding-paragraph">
+							{ preventWidows(
+								translate(
+									'WordPress.com Payments makes it easy to sell physical and digital goods, accept donations, charge for in-person services, build subscription newsletters, and more.'
+								)
+							) }
+						</p>
+						<p className="memberships__onboarding-paragraph">
+							{ preventWidows(
+								translate(
+									'One-time, monthly, and yearly credit and debit card payment options are supported. {{link}}Learn more about payments.{{/link}}',
```

Comment:
This can wait for a follow up, but it may make sense to show this via an InlineSupportLink (similar to what we do for educational cards).